lib/disasters.js

```diff
@@ -1,23 +1,36 @@
 "use strict";
-var log = require('../log'),
-  models = require('../models'),
+ var models = require('../models'),
   async = require('async'),
   restify = require('restify'),
-  _ = require('lodash');
+  _ = require('lodash'),
+  Contact = require('../models').Contact,
+  models  = require('../models'),
+  mail = require('../mail'),
+  request = require('request'),
+    disasters = require('./disasters'),
+    _ = require('lodash'),
+  restify = require('restify');
```

Comment:
Restify is required 2 times, why do you include "disasters" since I believe it's the current file, and why do you include "models" ?